openerp/openupgrade/doc/source/modules80-90.rst

```diff
@@ -15,542 +15,657 @@ Status :
-|website_sale                       | Done                              |
-+-----------------------------------+-----------------------------------+
-|website_sale_delivery              |                                   |
-+-----------------------------------+-----------------------------------+
-|website_sale_digital               |                                   |
-+-----------------------------------+-----------------------------------+
-|website_sale_options               |                                   |
-+-----------------------------------+-----------------------------------+
-|website_sale_stock                 |                                   |
-+-----------------------------------+-----------------------------------+
-|website_slides                     |                                   |
-+-----------------------------------+-----------------------------------+
-|website_twitter                    |                                   |
-+-----------------------------------+-----------------------------------+
-|web_tip                            | Nothing to do                     |
-+-----------------------------------+-----------------------------------+
-|web_view_editor                    | Nothing to do                     |
-+-----------------------------------+-----------------------------------+
+Modules that are new in this release are marked with |new|. Modules that are
+missing in the new release are marked with |del|.
+
+.. |new| image:: images/new.png
+.. |del| image:: images/deleted.png
+
++----------------------------------------+-----------------------------------+
+|Module                                  |Status                             |
++========================================+===================================+
+|account                                 | Done (tentative)                  |
++----------------------------------------+-----------------------------------+
+|account_accountant                      | Nothing to do                     |
++----------------------------------------+-----------------------------------+
+| |del| account_analytic_analysis        |                                   |
++----------------------------------------+-----------------------------------+
+|account_analytic_default                | Nothing to do                     |
++----------------------------------------+-----------------------------------+
+| |del| account_analytic_plans           |                                   |
++----------------------------------------+-----------------------------------+
+| |del| account_anglo_saxon              |                                   |
++----------------------------------------+-----------------------------------+
+|account_asset                           | Done                              |
++----------------------------------------+-----------------------------------+
+| |del| account_bank_statement_extensions|                                   |
```

Comment:
As you are changing the entire file, better to give some more room to the columns, specially the second column that can contain references to renamings